alpheios_nemo_ui/data/assets/scss/breadcumb-navbar.scss

```diff
@@ -0,0 +1,86 @@
+.breadcumb-navbar {
+    overflow: hidden;
+    display: block;
+    text-align: center;
+
+    .nav-link-current-text {
+        font-size: 14px;
```

Comment:
can we use variables here? Style guide defines large label as 14px regular sans and then says that this text should be label label on Mobile and H4 on Desktop. So I think we need a new $label-large variable, and then we could have the base style for mobile be :  `font-size: $label-large; font-family: $font-family-sans;` and  override for media-breakpoint-up-lg to `font-size: $h4-font-size; font-family $font-family-serif`